luci-app-overthebox/luasrc/controller/admin/overthebox.lua

```diff
@@ -132,7 +132,7 @@ function interfaces_status()
 	end
 	-- Check overthebox service are running
 	mArray.overthebox["tun_service"] = false
-	if string.find(sys.exec("/usr/bin/pgrep -f ^glorytun"), "%d+") then
+	if string.find(sys.exec("/usr/bin/pgrep -f 'glorytun-(tcp|udp)'"), "%d+") then
```

Comment:
This will also return process id for toto-glorytun-udp-test.sh
Is it really what you want ?